pkgs/development/interpreters/python/default.nix

```diff
@@ -46,7 +46,7 @@ with pkgs;
         inherit hasDistutilsCxxPatch pythonForBuild;
   };
 
-in {
+in rec {
```

Comment:
Why was this `rec` added? It doesn't seem to be necessary and breaks overriding/adding packages in the Python package set in an overlay using `packageOverrides`. The overrides do not end up applied to `python3Packages.python.pkgs`.

Reply:
cc @FRidh

Reply:
removed rec in 92d2153e6c54439fdc3e822a10c54a1b47fd32f4. If overriding is broken, show it with an example.

Reply:
Thanks, that fixes it.